app/adminPage.html

```diff
@@ -91,6 +93,17 @@ <h4 class="search-heading">Filter Results</h4>
                     <tbody class="score_list user-table"></tbody>
                 </table>
             </div>
+            <form class="new-test-container" id="testForm" method="post">
+                <h2>Add new test</h2>
+                <input name="testName" type="text" id="testName" placeholder="Test name">
+                <input type="submit" class="btn primary-button new-test-button submit" value="Add new test">
+                <div id="message-target"></div>
+                <div class="buttons">
```

Comment:
buttons seems like a bad class name here, it doesnt contain any buttons?

Reply:
Removed the buttons div and the messageArea div as they were not doing anything and we not referenced in other parts of the application.